docs/docs/deploying-to-azure.md

```diff
@@ -0,0 +1,136 @@
+
+1. Finally, add the change and commit it.
+
+   ```bash
+   git add .
+   git commit -m "adding Gatsby project"
+   ```
+
+### Create a GitHub repository and push your code
+
+1. Go to GitHub and log in. You should now be at a URL like `https://github.com/<your username>?tab=repositories`.
+
+2. Click the `new` button.
+
+3. Name your repository and click `Create repository`.
+
+4. Finally, add your GitHub repository as a remote and push. Type the following commands to accomplish that (replacing `<user>` with your GitHub user name):
+
+   ```bash
+   git remote add origin https://github.com/<user>/gatsby-app.git
+   git push -u origin master
+   ```
+
+Now you are ready to deploy your site!
+
+## Deploy to Azure Static Web Apps
+
+1. Sign in to the [Azure portal](https://portal.azure.com/learn.docs.microsoft.com?azure-portal=true).
+1. In the top bar, search for **Static Web Apps**.
+1. Select **Static Web Apps**.
+1. Select **New**.
+
+### Configuring settings
+
+Next, configure your new app and link it to your GitHub repository.
+
+1. Enter the **Project Details**.
+
+   | Setting          | Value                                  |
+   | ---------------- | -------------------------------------- |
+   | _Subscription_   | **Your chosen subscription**           |
+   | _Resource Group_ | _A pre-existing or new resource group_ |
+
+1. Enter the **Static Web Apps details**.
+
+   | Setting  | Value                                                                         |
+   | -------- | ----------------------------------------------------------------------------- |
+   | _Name_   | Name your app. Valid characters are `a-z` (case insensitive), `0-9`, and `_`. |
+   | _Region_ | Select Region closest to you                                                  |
+   | _SKU_    | **Free**                                                                      |
+
+1. Click the **Sign-in with GitHub** button and authenticate with GitHub.
+1. Enter the **Source Control Details**.
+
+   | Setting        | Value                                                    |
+   | -------------- | -------------------------------------------------------- |
+   | _Organization_ | Select the Organization where you created the repository |
+   | _Repository_   | **gatsby-app**                                           |
+   | _Branch_       | **master**                                               |
+
+1. Click the **Next: Build >** button to edit the build configuration.
+
+### Building your site
+
+Next, add configuration details specific to your preferred frontend framework.
+
+| Setting                 | Value         |
+| ----------------------- | ------------- |
+| _App location_          | _Leave blank_ |
+| _Api location_          | _Leave blank_ |
```

Comment:
These are `/` and `api` by default and "App location" is a required field. I think a little more explanation here would be helpful to readers but as long as they know they can leave the default values there, we're fine.